core/deployment/src/main/java/io/quarkus/deployment/steps/RegisterForReflectionBuildStep.java

```diff
@@ -52,6 +52,7 @@ public void build(CombinedIndexBuildItem combinedIndexBuildItem, Capabilities ca
             boolean fields = getBooleanValue(i, "fields");
             boolean ignoreNested = getBooleanValue(i, "ignoreNested");
             boolean serialization = i.value("serialization") != null && i.value("serialization").asBoolean();
+            boolean unsafeAllocated = i.value("unsafeAllocated") != null && i.value("unsafeAllocated").asBoolean();
```

Comment:
Shouldn't a `boolean unsafeAllocated()` method exist in  https://github.com/quarkusio/quarkus/blob/main/core/runtime/src/main/java/io/quarkus/runtime/annotations/RegisterForReflection.java?

Reply:
Yes it should! Thanks for catching this